src/scss/mixins/_date.scss

```diff
@@ -0,0 +1,33 @@
+@mixin oFormsDateFeatures($class) {
+    .#{$class}--date .#{$class}__group {
+        // We want to use `box-sizing: border-box` to set a max-height which includes padding.
+        // However when setting width to an `ex` value we want padding to be in addition.
+        $padding-width: $_o-forms-field-default-padding-leftright * 2;
+        margin-top: oTypographySpacingSize(2);
+
+        .#{$class}__label {
+            @include oFormsGroupLabel();
+            margin-bottom: oTypographySpacingSize(1);
```

Comment:
Also a pedantic observation: some properties are assigned variable values, others are using this o-type function

Reply:
Hopefully the distinction is variables for shared values and o-type directly for non-shared values. Whether this is true or not I'm unsure. 🙈Maybe something for a different PR.

Reply:
If the distinction is meant to be implied, I think we should probably document that somewhere because we're not too consistent about it 😬

Reply:
Yeah good shout. There're an awful lot of spacing variables already. Rationalising is something for another PR though... It would be great to do a breaking of o-forms and pull out some things into their own components to simplify things, in my opinion.